docs/website/02_examples/vonMisesTruss.md

```diff
@@ -7,13 +7,13 @@ standard Von-Mises truss example found in literature (refer to Section 2[@misesT
 
 ## Code highlights
 
-The struct named `Truss` is created such that it inherits from `PowerBasisFE`.
+The struct named `Truss` is created such that it inherits from `FEBase`.
 It must be decorated with `AutoDiffFE` as well to compute the stiffness matrix and load vectors during construction.
 It is constructed as shown below:
 
 ```cpp
 Truss(const Basis &basis, const typename LocalView::Element &element, double p_EA)
-    : BaseDisp(basis, element), EA{p_EA} {
+    : FEBase<Basis, useFlat>(basis, element), EA{p_EA} {
```

Comment:
Maybe FEBase should have true as default and then here only FEBase<Basis> will be left. Otherwise this random name `useFlat`  is confusing here.

Reply:
I thought in the future, may be it is nicer to have the flexibility of using either the flat or the untouched version of the basis from the finite element side, and hence I am passing it explicitly from the finite element and did not use the default provided in `FEBase`. Else, the non-type template paramater `useFlat` is not needed for the finite elements at all and it will always assume `FlatBasis`. 
In a separate PR, may be then `using FlatBasis = typename Traits::FlatBasis;` can also be replaced to `using BasisVersion= typename Traits::BasisVersion;`
What do you think?

Reply:
Yes you are right but I'm concert that this makes our documentation confusing. Maybe you can refactor such that:
E.g. 
```cpp
template<tyname Basis, bool useFlat>
class Truss: FEBase<Basis, useFlat>
{
using Base = FEBase<Basis, useFlat>;

Truss(const Basis &basis, const typename LocalView::Element &element, double p_EA)
    : Base(basis, element), EA{p_EA} {
...
}
```

Then the constructor looks nicer

Reply:
Ah okay. I will then fix it in `ikarus-examples` and amend the corresponding documentation. I think the finite elements in `ikarus` can stay as such.

Reply:
Ok for me